README.md

```diff
@@ -61,12 +61,49 @@ If are using this module in a browser with something like browserify, then you w
 $ ipfs config --json API.HTTPHeaders.Access-Control-Allow-Origin "[\"http://example.com\"]"
 ```
 
-## API
+## Usage
 
-### Level 1 Commands
-Level 1 commands are simple commands
+We classify the API calls by 'core', 'extensions', 'tooling', and 'network', following the same API spec organization available at [ipfs/specs](https://github.com/ipfs/specs/blob/ipfs/api/api/core/README.md)
 
-#### add
+The tests folder also contains great examples that can be used to understand how this client library interacts with the HTTP-API. You can find the tests here: https://github.com/ipfs/js-ipfs-api/tree/master/test/api
+
+### Core
```

Comment:
I don't understand what comes below, why are you adding `curl`in here instead of examples of the js-ipfs-api usage? Also what does the empty `Block` mean?